classes/PodsUI.php

```diff
@@ -3686,7 +3828,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( empty( $value ) ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 548087a89416638c1289bfc3c6c723d4</sub>